src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/rpa/MoreTimeRequestedNotificationServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.cmc.claimstore.rpa;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.cmc.claimstore.MockSpringTest;
+import uk.gov.hmcts.cmc.claimstore.events.response.MoreTimeRequestedEvent;
+import uk.gov.hmcts.cmc.claimstore.rpa.config.EmailProperties;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.cmc.claimstore.rpa.ClaimIssuedNotificationService.JSON_EXTENSION;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildJsonMoreTimeRequestedFileBaseName;
+
+public class MoreTimeRequestedNotificationServiceTest extends MockSpringTest {
+
+    @Autowired
+    private MoreTimeRequestedNotificationService service;
+    @Autowired
+    private EmailProperties emailProperties;
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
+
+    @Captor
+    private ArgumentCaptor<EmailData> emailDataArgument;
+
+    private Claim claim;
+
+    private MoreTimeRequestedEvent event;
+
+    @Before
+    public void setUp() {
+        claim = SampleClaim
```

Comment:
Please make claim local variable. No reason to make it class field.

Reply:
claim is required for more-time-req event which is used in both tests.